src/components/app/Initializing.js

```diff
@@ -7,7 +7,7 @@ import React from 'react';
 
 const PROFILER_URL = 'https://github.com/firefox-devtools/Gecko-Profiler-Addon';
 
-export default function Initializing() {
```

Comment:
This is fine to leave in here, but in the future it would be good to have code changes only relevant to the current PR. They are unrelated to the current patch, and make the review process a bit harder. It's perfectly fine to send in smaller PRs. They are fast to review, and merge in.